youtube_dl/utils.py

```diff
@@ -2371,15 +2371,12 @@ def set_alpn_protocols(ctx):
         return YoutubeDLHTTPSHandler(params, context=context, **kwargs)
 
 
-def bug_reports_message():
+def bug_reports_message(before=';'):
     if ytdl_is_updateable():
         update_cmd = 'type  youtube-dl -U  to update'
     else:
         update_cmd = 'see  https://yt-dl.org/update  on how to update'
```

Comment:
Revert and fix `utils.bug_reports_message()` (2a):
```suggestion
        update_cmd = 'see  https://github.com/ytdl-org/youtube-dl/#user-content-installation  on how to update'
```